static/scss/answers/interactive-map/VerticalFullPageMap.scss

```diff
@@ -562,9 +562,8 @@
     }
 
     &-closeCardButtonWrapper {
```

Comment:
Instead of changing the close card button, could we use target the styling for the title itself? Potentially, could we use `overflow-wrap: break-word` or something like that? It feels a bit strange that to fix the title overflow issue we're altering the styling of things around the title.

Reply:
The root problem is that the closeCardButtonWrapper is using `position: absolute` which takes the div out of the document's normal flow, so the adjacent `ordinalAndTitle` div will expand right into it. Even if we set `overflow-wrap: break-word`, the title will overflow because the page will act as if the `closeCardButtonWrapper` div isn't even there.

Reply:
Got it!